### DISCUSSION TITLE
Where is the result from Agent GPT?

### DISCUSSION THREAD

**Discussion by Nick390** [reworkd/AgentGPT#203] [Q&A] [answered]
I asked it to program a website and it continued to write a lot of words, and it told me that it would do a specific feature and so on, but in the end it stopped and showed this message
```
This agent has been running for too long (25 Loops). To save your wallet, and our infrastructure costs, this agent is shutting down. In the future, the number of iterations will be configurable.
``` 
The question is where is the site code that is supposed to provide it to me?

Data produced
[AgentGPT.txt](https://github.com/reworkd/AgentGPT/files/11240514/AgentGPT.txt)

**Comment by aiastia** [comment]
+1

**Comment by mentorwan** [comment]
+1

**Comment by liqingjun123** [comment]
+1

**Comment by mosezh** [comment]
me too

**Comment by ahmedaizi** [comment, marked as answer]
A copy and paste from reddit where a user had the same problem:

"Thanks for the feedback and thanks for trying it out! Currently it doesn't have the "arms" and "legs" to actually go about making those changes. This current beta is just there to demonstrate what's possible. We're working hard to actually allow AgentGPT to interface with tools such as: web search, code editor, documents, etc. Stay tuned!"

**Reply by T-Quirk** [reply]
Hi,

Just to confirm if asked to create a programme the agent wont actually create it (yet) it will just lay out the steps as if it were creating the programme? Please let me know that this is correct. If not, please educate me on where i can find the finished product.

Thanks

**Reply by vdruts** [reply]
Yes it will lay out the steps and then hallucinate all of the output.

**Comment by APiTJLillo** [comment]
Converting to discussion, feel free to continue the discussion there if needed.

**Comment by ahmedaizi** [comment]
Yeah, that’s correct, unfortunately it won’t actually create it at the
moment.

On Mon, 17 Apr 2023 at 22:49, TQuirk ***@***.***> wrote:

> Hi,
>
> Just to confirm if asked to create a programme the agent wont actually
> create it (yet) it will just lay out the steps as if it were creating the
> programme? Please let me know that this is correct. If not, please educate
> me on where i can find the finished product.
>
> Thanks
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/reworkd/AgentGPT/discussions/203#discussioncomment-5641255>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AY3TAVT2M77V3LK2A5YERQLXBWNDJANCNFSM6AAAAAAXBUPAJM>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
Ahmed Aizi
Medical Student
University of Medicine and Pharmacy "Iuliu Hatieganu" Cluj Napoca
+40 ***@***.***

**Comment by iTzTruth** [comment]
I am really suprised this is popular, its literally nothing but chatting with itself..... creating nothing..... 

Why is it popular, at this point its worse than any ai on the market, it literally does NOTHING... Just loops

**Reply by vdruts** [reply]
Welcome to the hype-train.

**Reply by SOS-Recon** [reply]
Same frustration. Had no idea that it wasn't actually going to give me anything in return for allowing it to tap into my OpenAI API and burn cycles and $1.80 in credits. No big loss, but maybe a disclaimer stating that, you know, it wasn't actually going to produce real results.
